examples/async-sign/signed_message/src/lib.rs

```diff
@@ -0,0 +1,89 @@
+#![no_std]
+
+use codec::Decode;
+use gstd::{debug, exec, msg, prelude::*, ProgramId};
+use scale_info::TypeInfo;
+use sp_core::{
+    crypto::{AccountId32, UncheckedFrom},
+    sr25519,
```

Comment:
We can import 'Pair`, `Public`, `Signature` here to be shorter below.

```suggestion
    sr25519::{Pair, Public, Signature},
```

Reply:
ok